lerna.json

```diff
@@ -0,0 +1,16 @@
+{
+  "lerna": "2.0.0-beta.30",
+  "version": "0.3.2",
+  "changelog": {
+    "repo": "apollostack/apollo-server",
+    "labels": {
```

Comment:
@DxCx  I might be able to create the labels.  Is creating a changelog a feature of lerna?